formats/coming_soon/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "replaced_content_item"
+  ],
+  "properties": {
+    "replaced_content_item": {
```

Comment:
I understand we want consistency, but this name is confusing for coming_soon items because they are _replaced by_ the item that I think this will link to.